styles/dowhop-profile.css

```diff
@@ -258,3 +258,15 @@ svg.social-media-svg {
 #profile-about {
   font-family: inherit;
 }
+
+.about {
+  margin-left: 55px;
+  text-align: center;
+}
+.form {
```

Comment:
In the future, you might want to avoid using class names that are the same as element names - it could get confusing! 

One thing I like to do is use prefixes, like `profile-form`, etc.